cheerio-scraper/INPUT_SCHEMA.json

```diff
@@ -17,7 +17,8 @@
 			"type": "boolean",
 			"description": "If enabled, the scraper will support adding new URLs to scrape on the fly, either using the <b>Link selector</b> and <b>Pseudo-URLs</b> options or by calling <code>context.enqueueRequest()</code> inside the <b>Page function</b>. Use of the request queue has some overheads, so only enable this option if you need to add URLs dynamically.",
 			"default": true,
-			"groupCaption": "Options"
+			"groupCaption": "Options",
+			"editor": "hidden"
```

Comment:
Did you check how it looks in the UI? I'm a bit afraid that the `groupCaption` together with `hidden` will cause some problems.

Let's move the `groupCaption` to `keepUrlFragments` and move the `useRequestQueue` to the end of the input schema.

Reply:
Also, let's remove the `default: true` so that later, `useRequestQueue` can be fully removed from the schema.